iree/compiler/Dialect/LinalgPlus/IR/LinalgPlusOps.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Dialect/LinalgPlus/IR/LinalgPlusOps.h"
+
+#include "iree/compiler/Dialect/LinalgPlus/IR/LinalgPlusDialect.h"
+#include "llvm/ADT/STLExtras.h"
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/Support/SMLoc.h"
+#include "mlir/IR/Attributes.h"
+#include "mlir/IR/Builders.h"
+#include "mlir/IR/Diagnostics.h"
+#include "mlir/IR/OpImplementation.h"
+#include "mlir/IR/OperationSupport.h"
+#include "mlir/IR/PatternMatch.h"
+#include "mlir/IR/TypeUtilities.h"
+#include "mlir/IR/Value.h"
+#include "mlir/Support/LLVM.h"
+#include "mlir/Support/LogicalResult.h"
+
+namespace mlir {
+namespace iree_compiler {
+namespace linalg_plus {
+
+void SortOp::build(OpBuilder &builder, OperationState &state,
```

Comment:
Instead of this we could just call the auto-generated build method. We just need to pass the types based on the types of the operands.

Reply:
Wouldnt this work 

```
auto types = llvm::to_vector<4>(llvm::map_range(operands, [&](Value operand) { return operand.getType(); }));
build(builder, state, types, operands, dimension);
```

I am assuming the auto-generated method already adds region?

Reply:
I deleted the build method, and think that we can add it when needed.